Makefile

```diff
@@ -154,6 +154,10 @@ ifdef use
   endif
 endif
 
+ifeq ($(usedbg),true)
+  dbgcmd := lldb --batch --one-line "breakpoint set --name main" --one-line run --one-line "process handle SIGINT --pass true --stop false" --one-line "process handle SIGUSR2 --pass true --stop false"  --one-line "thread continue" --one-line-on-crash "frame variable" --one-line-on-crash "register read" --one-line-on-crash "bt all" --one-line-on-crash "quit 1" --
```

Comment:
Can I suggest a different name for this flag?

I think that `dbg` or `debug` as terms are confusing with regard to release vs debug mode.

Can I suggest renaming as `uselldb` and `lldbcmd`?

Reply:
updated